<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>baggeocoder_form</class>
 <widget class="QDialog" name="baggeocoder_form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>450</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Geocode with Google</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>410</y>
     <width>160</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_opslaan_als">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Opslaan als Shapefile:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="shapefilename">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>geocoded.shp</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_shapefile">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>79</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Kies...</string>
   </property>
  </widget>
  <widget class="QComboBox" name="woonplaats">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>138</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="adres">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>108</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="adres_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>112</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Adres (met huisnr) of Postcode</string>
   </property>
  </widget>
  <widget class="QLabel" name="selecteer_bestand_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecteer CSV bestand:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="infilename">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_infile">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>79</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Kies...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_niet_gevonden">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Niet gevonden locaties CSV bestand:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="notfoundfilename">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>niet_gevonden.csv</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_notfound">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>79</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Kies...</string>
   </property>
  </widget>
  <widget class="QComboBox" name="huisnummer">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="huisnummer_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>192</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Huisnummer (1,2,3,...)</string>
   </property>
  </widget>
  <widget class="QLabel" name="huisnummer_toevoeging_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>222</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Huisnummer toevoeging (H,1,2..)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="huisnummer_toevoeging">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="koppel_kolomvelden_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Koppel de kolomvelden voor de zoek query: </string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_kolom">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>165</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>kies kolom</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_lijst">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>142</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>kies uit lijst</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="woonplaats_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Woonplaats</string>
   </property>
  </widget>
  <widget class="QLabel" name="huisnummer_letter_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>252</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Huisnummer letter (A-Z)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="huisnummer_letter">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>286</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>baggeocoder_form</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>baggeocoder_form</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
